podaac/swodlr_ingest_to_sds/submit_to_sds.py

```diff
@@ -86,8 +86,8 @@ def lambda_handler(event, _context):
 
 def _parse_record(record):
     cmr_r_message = json.loads(record['body'])
-    identifier = cmr_r_message['identifier']
     filename, s3_url = _extract_s3_url(cmr_r_message)
+    identifier = filename.split('.', 1)[0]
```

Comment:
If you're trying to split off the extension from the filename I would use Path.stem https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.stem
Filename could include many `.`

Reply:
Nope - I just wanted the base filename which is the left-most string